src/openrct2/util/Util.cpp

```diff
@@ -27,6 +27,14 @@ float UtilRandNormalDistributed()
     return _distributor(_prng);
 }
 
+uint8_t UtilRandUniformDistributedByte()
+{
+    thread_local std::mt19937 _prng{ std::random_device{}() };
+    // char types are not permitted as type params of uniform_int_distribution...
+    thread_local std::uniform_int_distribution<uint16_t> _distributor{ 0, 255 };
+    return static_cast<uint8_t>(_distributor(_prng));
+}
+
```

Comment:
Move this into the Simplex class which is also the only class this will ever use this and it doesn't need to be thread_local lwhen its part of the class.